CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ title: Changelog
 * `<cc-input-text>`: set default font back to `--cc-ff-monospace` when the input contains tags (BREAKING CHANGE).
 * Introduce a new project file structure (BREAKING CHANGE).
 * all components: change `rem` units to `em` (BREAKING CHANGE).
+* `<cc-html-frame>`: change `title` prop to `iframeTitle` (`iframe-title` when used as an attribute) to avoid conflicts with the native `title` attribute (BREAKING CHANGE).
```

Comment:
@hsablonniere this is a breaking change but I figured it was ok to do it because it's only used in our `cc-invoice` component.
Do you know of any use outside this, for instance the admin console or the docs website ?

Reply:
:clap: Nice catch. I'm OK with this. No usage apart from invoice to the best of my knowledge.